radis/lbl/broadening.py

```diff
@@ -1195,6 +1297,70 @@ def _add_doppler_broadening_HWHM(self, df, Tgas):
 
         return
 
+
+    def voigt_lineshape(self,w_centered, hwhm_lorentz, hwhm_voigt, jit=True):
```

Comment:
Same, there is no reason to add them as class methods

Reply:
As I said in the PR Description : The changes in broadening.py and factory.py must be ignored. 
They come from the previous version I showed you during the Teams call and I didn't want to push them. Indeed, the doppler_kurucz method isn't used in my current code which only relies on SpectrumFactory as it is.
So basically the broadening methods are used as they are. 
One last thing about the partfcns : since I added the linestrength at the Tref temperature to the df early in the API in the read_kurucz method, the calc_linestrength_eq method ran with this int parameter and didn't force me to add a new Class for partfcns as we discussed it. However, if I must not add the int to the df and compute the linestrength with the Exojax Sij0 method, I will have to make changes to partfunctions.

Reply:
Please push now the revert of these files. Otherwise it's super hard for us to review